tests/test_core.py

```diff
@@ -679,11 +676,12 @@ def test_persistant_state(package):
 
 
 def test_persistant_state_integration(package):
-    if package.version == 7:
-        pytest.xfail(reason="MAGICC7 TCR/ECS diagnosis is currently broken")
-        package.diagnose_tcr_ecs()
     test_ecs = 1.75
     package.update_config(CORE_CLIMATESENSITIVITY=test_ecs)
+    if package.version == 7:
+        # ecs only follows its definition with IPCCTAR forcing (MAGICC7 bug...)
```

Comment:
I'm a bit worried that this might go out of date (once it's fixed) - is there an issue number which could be tracked (even if in private repo)?

Reply:
Ok I re-wrote the test to make it tracked here. See what you think